### PR TITLE
Bump Guava from 30.1.1 to 31.0.1

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

File: integration-tests/kafka-avro/pom.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>30.1.1-jre</version>
+            <version>31.0.1-jre</version>
```
Comment (famod) [inline integration-tests/kafka-avro/pom.xml:98]: Can please you check whether that version can be removed?

**Comment by famod** [comment]
@aloubyansky not sure we should bump this individually? Seems this should be in line with some other bootstrap deps?

**Comment by aloubyansky** [comment]
How critical this update is?

**Comment by nastra** [comment]
> How critical this update is?

It's not critical at all. Just figured it's worth updating now that there's a new version

**Comment by aloubyansky** [comment]
Thanks for the info @nastra Given that guava's version is primarily dictated by the thirdparty libraries Quarkus bootstrap depends on, as @famod said, we should not update it on its own unless there is a critical issue it fixes. I'll close this PR for now. Thanks for taking care though.

**Pull request closed without merging**